cl/recap/management/commands/pacer_iquery_scraper.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import print_function
+import sys
+
+from django.db import IntegrityError
+from django.db.models import Q, Count
+from lxml.etree import XMLSyntaxError
+
+from cl.lib.command_utils import VerboseCommand
+from cl.lib.db_tools import queryset_generator
+from cl.search.models import Docket
+from cl.alerts.models import DocketAlert
+from cl.alerts.tasks import send_docket_alert
+from datetime import datetime, timedelta
+
+
+class Command(VerboseCommand):
+    help = "Scrape PACER for all alerts"
+
+    def handle(self, *args, **options):
```

Comment:
In order to run as a daemon, this needs some kind of loop that keeps the function running non-stop. See scrape_rss.py for an example.